packages/0x.js/contract_templates/contract.mustache

```diff
@@ -2,6 +2,7 @@
  * This file is auto-generated using abi-gen. Don't edit directly.
  * Templates can be found at https://github.com/0xProject/0x.js/tree/development/packages/abi-gen-templates.
  */
+// tslint:disable-next-line:no-unused-variable
```

Comment:
We shouldn't be linting `.mustache` files.

Reply:
We're not. But generated files have unused import TxDataPayable if the generated contract doesn't have payable functions. It'll be too complicated to detect it so I just ignore it.

Reply:
Ah, I see!